server/src/main/java/org/apache/druid/client/HttpServerInventoryView.java

```diff
@@ -136,12 +135,12 @@ public void start()
         throw new ISE("can't start.");
       }
 
-      log.info("Starting HttpServerInventoryView.");
+      log.info("Starting %s.", execNamePrefix);
```

Comment:
if you are changing here, there are few more places in this class that need to be changed.